README.md

```diff
@@ -10,23 +10,24 @@ For more complex use cases, see the [setup-mabl-cli](https://github.com/marketpl
 ### Example workflow: Simple
 
 ```
+name: mabl test workflow
 on: [push]
 
-name: mabl Simple Example
-
 jobs:
   test:
     name: mabl Test
     runs-on: ubuntu-latest
+    environment: 
+      name: production
+      url: https://sandbox.mabl.com/
     steps:
       - uses: actions/checkout@v2
-
-      - name: Functional test deployment
+      - name: Trigger mabl Deployment Event
         id: mabl-test-deployment
         uses: mablhq/github-run-tests-action@v1
         env:
           MABL_API_KEY: ${{ secrets.MABL_API_KEY }}
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
This is the name of the [standard token provided by GitHub](https://docs.github.com/en/actions/security-guides/automatic-token-authentication#using-the-github_token-in-a-workflow). Do we really need to change this?